docs/en/observability/aws-deploy-elastic-serverless-forwarder.asciidoc

```diff
@@ -0,0 +1,317 @@
+  * `args.cloud_id`: Cloud ID of elasticsearch endpoint. Mandatory when `args.elasticsearch_url` is not provided. Will be ignored if `args.elasticsearch_url` is defined as well.
+  * `args.username`: Username of the elasticsearch instance to connect to. Mandatory when `args.api_key` is not provided. Will be ignored if `args.api_key` is defined as well.
+  * `args.password` Password of the elasticsearch instance to connect to. Mandatory when `args.api_key` is not provided. Will be ignored if `args.api_key` is defined as well.
+  * `args.api_key`:  API key of elasticsearch endpoint in the format **base64encode(api_key_id:api_key_secret)**. Mandatory when `args.username`  and `args.password` are not provided. Will take precedence over `args.username`/`args.password` if both are defined.
+  * `args.es_datastream_name`: Name of data stream or index where logs should be forwarded to. Lambda supports automatic routing of various {aws} service logs to the corresponding data streams for further processing and storage in the {es} cluster. It supports automatic routing of `aws.cloudtrail`, `aws.cloudwatch_logs`, `aws.elb_logs`, `aws.firewall_logs`, `aws.vpcflow`, and `aws.waf` logs. For other log types, if using data streams, you can optionally set its value in the configuration file according to the naming convention for data streams and available integrations. If the `es_datastream_name` is not specified and it cannot be matched with any of the above {aws} services, then the value will be set to `logs-generic-default`. In version **v0.29.1** and earlier, this configuration parameter was named `es_index_or_datastream_name`. Rename the configuration parameter to `es_datastream_name` in your `config.yaml` file on the S3 bucket to continue using it in the future version. The older name `es_index_or_datastream_name` is deprecated as of version **v0.30.0**. The related backward compatibility code is removed from version **v1.0.0**.
+  * `args.batch_max_actions`: Maximum number of actions to send in a single bulk request. Default value: 500.
+  * `args.batch_max_bytes`: Maximum size in bytes to send in a single bulk request. Default value: 10485760 (10MB).
+  * `args.ssl_assert_fingerprint`: SSL fingerprint for self-signed SSL certificate on HTTPS transport.
+
+// is there some logic to this order? i.e. just placed ssl at end - should we re-order?
+
+[[aws-serverless-forwarder-deploy-sar]]
+== Deploy `elastic-serverless-forwarder` from Serverless Application Repository
+
+There are several deployment methods available via the {aws} Serverless Application Repository:
+
+* <<aws-serverless-forwarder-deploy-console>>
+* <<aws-serverless-forwarder-deploy-cloudformation>>
+* <<aws-serverless-forwarder-deploy-terraform>>
+
+[[aws-serverless-forwarder-deploy-console]]
+=== Deploy using {aws} Console
+
+. Log in to {aws} console and navigate to the Lambda service.
+. Click **Create function**.
```

Comment:
this changed a little in the current AWS console :(

Reply:
I ran thru it again and updated the text and screenshot